DOC-2411: Clearing the formula input and then saving didn't remove the element (#3298)
tinymce/tinymce-docs: modules/ROOT/pages/7.1.1-release-notes.adoc

The following premium plugin updates were released alongside {productname} {release-version}.

=== Math Plugin

The {productname} {release-version} release includes an accompanying release of the **Math** premium plugin.

> **Premium plugin Math**, includes the following fix.

Reviewer: ```suggestion
The **Math** premium plugin includes the following fix:
```

==== Clearing the formula input and then saving didn't remove the element.

> In the initial release of the Math Plugin, {productname} did nothing when the user saved an empty formula, even if a formula was selected.

Reviewer: ```suggestion
In the initial release of the Math Plugin, {productname} did not remove the selected formula and element when the user saved an empty formula.
```

> As a consequence, when the user cleared the formula and then clicked "save", {productname} did not remove the element from the document.

Reviewer: ```suggestion
As a consequence, when the user cleared the formula and clicked "save", the selected formula and element remained in the editor.
```

> {productname} {release-version} addresses this issue. Now, when a user has a formula selected, clears it, and then clicks "save", the selected formula and element is removed from the editor.

Reviewer: ```suggestion
{productname} {release-version} addresses this issue. Now, when a user has a formula selected, clears it, and clicks "save", the selected formula and element are properly removed from the editor.
```
